rskj-core/src/main/java/co/rsk/peg/Bridge.java

```diff
@@ -81,6 +83,7 @@
     public static final CallTransaction.Function GET_FEDERATION_ADDRESS = CallTransaction.Function.fromSignature("getFederationAddress", new String[]{}, new String[]{"string"});
     // Returns the minimum amount of satoshis a user should send to the federation.
     public static final CallTransaction.Function GET_MINIMUM_LOCK_TX_VALUE = CallTransaction.Function.fromSignature("getMinimumLockTxValue", new String[]{}, new String[]{"int"});
+    public static final DataWord RELEASE_BTC_TOPIC = new DataWord("release_btc_topic".getBytes(StandardCharsets.UTF_8));
```

Comment:
Do we want to use variable-length character encodings such as `UTF-8` for things we store in the blockchain? Or maybe  fixed-length (but obviously more restrictive) encodings such as `US_ASCII`?

Reply:
What would be the issue on storing variable-length characters in the blockchain?

Reply:
We'd have to be a bit more careful with the 32 byte limit for data words.

Reply:
That said, UTF-8 is broadly used in the node for encoding of strings, but I thought it might be a point to bring into discussion.

Reply:
For ascii characters there's no difference

Reply:
I know, that's why I suggest `US_ASCII` as a safe alternative to `UTF-8` in case of typos.